plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -49,7 +41,10 @@ import org.jetbrains.kotlin.utils.addToStdlib.ifTrue
  */
 class ProgramConverter(val session: FirSession, override val config: PluginConfiguration, override val errorCollector: ErrorCollector) :
     ProgramConversionContext {
-    private val methods: MutableMap<MangledName, FunctionEmbedding> = SpecialKotlinFunctions.byName.toMutableMap()
+    private val methods: MutableMap<MangledName, FunctionEmbedding> =
+        SpecialKotlinFunctions.byName.toMutableMap().also {
```

Comment:
Not super important, but it's a bit weird that we are making a map, then turning it into a mutable map, and then mutating it again.  If this is what we'll do with `SpecialKotlinFunctions.byName` anyway, maybe let's have it take a mutable map/mutable map builder?  That would also make it clear it is stateless.